datasets/hail-vep-pipeline.yml

```diff
@@ -0,0 +1,33 @@
+Name: Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin
+Description: VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release includes The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.
+Documentation: https://privoit.github.io/hail-vep-pipeline/
+Contact: https://www.privoit.com/contact-us
+ManagedBy: "[Privo](https://www.privoit.com/)"
+UpdateFrequency: New packages are added as soon as they are available and confirmed to work with recent versions of Hail.
+Tags:
+  - genome-wide association study
```

Comment:
1. Add - aws-pds to tags
2. Revise genome-wide association study to genome wide association study

I added vep and loftee to tags.yaml preemptively

Reply:
Added aws-pds tag and updated to remove hyphen from genome wide association study